pallets/rc-migrator/src/types.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Types
+
+use super::*;
+
+/// Relay Chain Freeze Reason
+#[derive(Encode, Decode)]
+pub enum AssetHubPalletConfig<T: Config> {
+	#[codec(index = 255)]
+	AhmController(AhMigratorCall<T>),
+}
+
+/// Call encoding for the calls needed from the Broker pallet.
+#[derive(Encode, Decode)]
+pub enum AhMigratorCall<T: Config> {
+	#[codec(index = 0)]
+	ReceiveAccounts { accounts: Vec<accounts::AccountFor<T>> },
+}
+
+/// Copy of `ParaInfo` type from `paras_registrar` pallet.
```

Comment:
```suggestion
/// Copy of `ParaInfo` type from `paras_registrar` pallet.
///
/// From: https://github.com/paritytech/polkadot-sdk/blob/b7afe48ed0bfef30836e7ca6359c2d8bb594d16e/polkadot/runtime/common/src/paras_registrar/mod.rs#L50-L59
```

Need to ensure that nobody changes this stuff in the meantime.

I think we should always put links here to the exact location that we copied, so that it is easier to check later.

Reply:
Fixed